### ISSUE TITLE
V19c: NEI lookup freeze with ChromatiCraft and GT installed together

### ISSUE THREAD

**Issue by FuYang001** [ReikaKalseki/Reika_Mods_Issues#1784]
NEI lookup freezed when I was using the newest release ChromatiCraft and GT5U.
Anything I looked up making game freezed.
At first, I thought that's DragonAPI's problem. But after I tested several other combinations:
DragonAPI+Gregtech √
DragonAPI+Gregtech+RoraryCraft+ReactorCraft+ElectriCraft √
DragonAPI+Gregtech+ChromatiCraft ×
Reika, could you please fix this? I do want to play with GT5U and CC.

**Comment by FuYang001** [comment]
Sorry, I waited 5 minutes and game ran again.

But I found a new problem that when I used "U" on a item of RoC, ReC, ElC and CC to check what can be crafted, many of them would spend at least 1 minute to wait. So I enabled the Debug module then got some information.
By using "R" to check recipes how to craft a item is normally displayed.
latest.log:https://pastebin.com/B1rThB46
logs: https://www.dropbox.com/s/f4ejo087rtgm79v/fml-client-latest.log?dl=0

The second time I spent 25 minutes getting freezed after I used "R".
